docs/src/man/comparisons.md

```diff
@@ -1,77 +1,173 @@
+|:-----------------------------|:------------------------------------------------------------------------------------------|:--------------------------------------------|
+| Aggregate by groups          | `df.groupby('grp')['x'].mean().reset_index()`                                             | `combine(groupby(df, :grp), :x => mean)`    |
+| Aggregate and add columns    | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')`                       | `transform(groupby(df, :grp), :x => mean)`  |
+| Aggregate and select columns | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')[['grp','x_mean','y']]` | `select(groupby(df, :grp), :x => mean, :y)` |
+
+### More advanced commands
+
+| Operation                 | pandas                                                                       | DataFrames.jl                                                 |
+|:--------------------------|:-----------------------------------------------------------------------------|:--------------------------------------------------------------|
+| Complex Function          | `df[['z']].agg(lambda v: np.mean(np.cos(v)))`                                | `combine(df, :z => (v -> mean(cos, skipmissing(v))))`         |
+|                           | `df['z'].mean()`                                                             | `combine(df, :z => (v -> mean(skipmissing(v))))` |
+| Transform several columns | `df.agg({'x': max, 'y': min})`                                               | `combine(df, :x => maximum, :y => minimum)`                   |
+|                           | `df[['x','y']].mean()`                                                       | `combine(df, [:x, :y] .=> mean)`                              |
+|                           | `df.filter(regex=("^x")).mean()`                                             | `combine(df, r"^x" .=> mean)`                                 |
+|                           | `df[['x', 'y']].agg([max, min])`                                             | `combine(df, ([:x, :y] .=> [maximum minimum])...)`            |
+| Multivariate function     | `df.assign(x_y_cor = np.corrcoef(df.x, df.y)[0,1])`                          | `transform(df, [:x, :y] => cor)`                              |
+| Row-wise                  | `df.assign(x_y_min = df.apply(lambda v: min(v.x, v.y), axis=1))`             | `transform(df, [:x, :y] => ByRow(min))`                       |
+|                           | `df.assign(x_y_argmax = df.apply(lambda v: df.columns[v.argmax()], axis=1))` | `transform(df, AsTable([:x,:y]) => ByRow(argmax))`            |
+| DataFrame as input        | `df.groupby('grp').head(2)`                                                  | `combine(d -> first(d, 2), groupby(df, :grp))`                |
+| DataFrame as output       | `df[['x']].agg(['max','min'])`                                               | `combine(:x => x -> (value = [minimum(x), maximum(x)],), df)` |
```

Comment:
inconsistent order of rows

Reply:
I have fixed the order.

I just realized that this is probably not a good example. The pandas version is just aggregation but since it turns into series so the shape ends up looking like a data frame.

Reply:
Maybe it's ok:
```
>>> df[['x']].agg(['min','max'])
     x
min  1
max  6
```

vs.

```
julia> combine(:x => x -> (value = [minimum(x), maximum(x)],), df)
2×1 DataFrame
│ Row │ value │
│     │ Int64 │
├─────┼───────┤
│ 1   │ 1     │
│ 2   │ 6     │
```